docs/posts/flink-unified-stream-batch/index.qmd

```diff
@@ -0,0 +1,414 @@
+---
+title: "Stream-batch unification through Ibis lens"
```

Comment:
I considered titling this "ELI5: How Ibis unifies streaming and batch"... LOL Perhaps there is a better title